.github/workflows/stale_community_prs.yml

```diff
@@ -1,6 +1,9 @@
 name: 'Check stale community PRs.'
 on: workflow_dispatch
       
+permissions:
+  pull-requests: write # to close stale PRs (actions/stale)
```

Comment:
This would also require `contents: read` but I think we've never run this workflow, so maybe we can remove it?